py/picca/io.py

```diff
@@ -7,6 +7,7 @@
 import time
 import os.path
 
+import constants
```

Comment:
this is unused

Reply:
and it should have beed `from picca import constants`

Reply:
I should have commented sorry, and I 'll modify that, thanks ! We won't actually need it once the psf from the mock_1D is in pixels (instead of angstrom), the modification has not been done yet so I am using the constant package at the line 304 that I commented for the push